src/components/ui/icon-devicon-plain-python.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from "react";
+
+// By: devicon-plain
```

Comment:
we could just add https://github.com/devicons/devicon/ as a dependency? im sure the other icons will be helpful to have in the future

Reply:
just realised react-icons/ai had a python icon - using that